src/Signature/Signature.js

```diff
@@ -160,13 +158,7 @@ const Signature = (props) => {
                 'vertical-align': 'middle',
               }}
             >
-              <img
-                width="200"
-                height="30"
-                style={{ width: '200px', height: '30px', maxWidth: 'none' }}
-                src={brandLogo.link}
-                alt={brandLogo.alt}
-              />
+              <img src={brandLogo.link} alt={brandLogo.alt} />
```

Comment:
We probably want width and height, pretty sure that was deliberate for email clients (including style and explicit width/height).